project/src/main/java/com/google/sps/data/VolunteeringOpportunity.java

```diff
@@ -0,0 +1,112 @@
+package com.google.sps.data;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class VolunteeringOpportunity {
```

Comment:
can you add an @override toString() function as well

Reply:
Added toString()